project.py

```diff
@@ -0,0 +1,153 @@
+from bs4 import BeautifulSoup
+import requests
+import re
+import pickle
+import os.path
+from pathlib import Path
+import random
+
+def dict_creation_one(script, things = dict()):
+    '''Create the base dictionary for ONE word; only to be called if there is no dicts1.p already'''
+    entiredict =  things
+    everything = script
+    newstring = ''
+    for x in script: #get rid of symbols that I don't want in my speech
+        if x not in ('[', ']', '@', '#', '$', '%', '^', '&', '*', '(', ')', ':', '"', '\\', '—', '|', '-'):
+            newstring += (x)
+    newstring = newstring.lower()
+    listowords = newstring.split()
+    index = 0
+    for y in listowords:    #run through all of the words and create a dictionary with them
+        index += 1
+        if index != len(listowords):
+            if y not in entiredict:
+                entiredict[y] = [listowords[index]]
+            else:
+                entiredict[y].append(listowords[index])
+    return entiredict
+
+def dict_creation_two(script, things = dict()):
+    '''Create the base dictionary for TWO words; only to be called if there is no dicts.p already'''
+    entiredict =  things
+    everything = script
+    newstring = ''
+    for x in script: #get rid of symbols that I don't want in my speech
+        if x not in ('[', ']', '@', '#', '$', '%', '^', '&', '*', '(', ')', ':', '"', '\\', '—', '|', '-'):
+            newstring += (x)
+    newstring = newstring.lower()
+    listowords = newstring.split()
+    index = 0
+    for x in range(0, len(listowords)-3): #run through all of the words and create a dictionary with them; the keys are tuples.
+        index = tuple((listowords[x], listowords[x+1]))
+        if index not in entiredict:
+            entiredict[index] = [listowords[x+2]]
+        else:
+            entiredict[index].append(listowords[x+2])
+    return entiredict
+
+def markov_chain_two(yup, yup2, allwordss):
+    """The markov chain function that actually generates the speech"""
+    spech = random.randrange(300, 500) #generates a random speech of length randomly selected in this range
+    speech = []
+    listowords = allwordss.split()
+    speech.append(random.choice(listowords).lower())
+    speech.append(random.choice(yup2[speech[0]]))
+    speech[0] = speech[0].capitalize()
+    for i in range(2, spech-1):
+        checker = (speech[i-2].lower(), speech[i-1].lower())
+        if '.' in (speech[i-1]) or ('?' in (speech[i-1])) or ('!' in (speech[i-1])):
+            speech.append(random.choice(yup[checker]).capitalize())
+        else:
+            appendix = random.choice(yup[checker])
+            if appendix in "i'm i i'll i've":
+                appendix = appendix.capitalize()
+            speech.append(appendix)
+    index = 0
+    for x in speech:        #A variety of statements that change Family Guy specific phrases to those that Trump could use
```

Comment:
Actually changing the words of the scripts is pretty important; I would have liked to see this in you writeup or in the docstring